migrations.go

```diff
@@ -14,13 +12,11 @@ import (
 
 func CheckMigration() error {
 	stump.Log("checking if repo migration is needed...")
-	p := util.IpfsDir()
-	oldverB, err := ioutil.ReadFile(filepath.Join(p, "version"))
+	oldver, err := GetCurrentVersion()
```

Comment:
agreed. we should have some function that maps ipfs version to a repo version for the versions we can't call `ipfs version --repo` on.